text-utility/text-utility.css

```diff
@@ -0,0 +1,62 @@
+.head {
+  color: #000;
+  word-break: break-all;
+}
+
+.h1-head {
+  font-size: 3rem;
+}
+
+.h2-head {
+  font-size: 2.5rem;
+}
+
+.h3-head {
+  font-size: 2rem;
+}
+
+.h4-head {
+  font-size: 1.75rem;
+}
+
+.h5-head {
+  font-size: 1.5rem;
+}
+
+.lg-text {
+  font-size: 1.5rem;
+  line-height: 1.5;
+  word-break: break-all;
```

Comment:
for word-break, you can a variable in typography use this property with the variable name like color.

Reply:
Yaa.. we could do that.. but as typography css is small in terms of code line.. so i thought to put it individually ..

Reply:
okay. you can merge.